pkg/operator/hive/hiveadmission.go

```diff
@@ -142,6 +142,7 @@ func (r *ReconcileHiveConfig) deployHiveAdmission(hLog log.FieldLogger, h resour
 
 	addConfigVolume(&hiveAdmDeployment.Spec.Template.Spec, managedDomainsConfigMapInfo, hiveAdmContainer)
 	addConfigVolume(&hiveAdmDeployment.Spec.Template.Spec, awsPrivateLinkConfigMapInfo, hiveAdmContainer)
+	addConfigVolume(&hiveAdmDeployment.Spec.Template.Spec, privateLinkConfigMapInfo, hiveAdmContainer)
```

Comment:
TODO: Adapt [webhook validation](https://github.com/openshift/hive/blob/0d5507f91935701146f3615c990941f24bd42fe1/pkg/validating-webhooks/hive/v1/clusterdeployment_validating_admission_hook.go#L351) to consume from the new configmap, when applicable. (And of course ultimately to do whatever necessary validation on the GCP incarnation of the config when it exists.)

And again, we _could_ decide to only add the configmap for the active controller here.